src/view/command_reference_sidebar.js

```diff
@@ -0,0 +1,61 @@
+/*
+ */
+
+var React = require("react");
+var Constants = require("../constants");
+var ResourceLoader = require("../util/resource_loader");
+var Sidebar = require('react-sidebar').default;
```

Comment:
Since you're adding a new dependency, you need to also update the "package.json" file. Please add a dependency on "react-sidebar" there (just like the others), with the exact version you have locally (I'm guessing it's 2.2.1, since it seems to be the latest). Then, run "npm install" just to check the syntax and everything is OK (it will complain if not).